src/api/constants.ts

```diff
@@ -20,13 +20,26 @@ export const APPLICATION = {
   },
 };
 
-export const PASSWORD = {
-  RANDOM_BYTES_SIZE: 32,
-  STRING_TYPE: 'hex',
-  PBKDF2: {
-    ITERATIONS: 10000,
-    KEY_LENGTH: 64,
-    DIGEST_ALGORITHM: 'sha512',
+export const ACCOUNT = {
+  EMAIL: {
+    VERIFICATION_EXPIRY: () => {
+      // 24 hours from now
+      const now = new Date();
+      const day = now.getDate();
+
```

Comment:
Do we need extra carriage return?

Reply:
As discussed, yes - this is "constants function" for generating a date (of 24 hours from now) so we return the date so we can consume it e.g `const date = CONSTANTS.EMAIL.VERIFICATION_EXPIRY();`